src/Scripts/Game/Features/LicensePlate/EL_LicensePlateGeneratorGeneric.c

```diff
@@ -5,9 +5,8 @@ class EL_LicensePlateGeneratorGeneric: EL_LicensePlateGeneratorBase
 	{
 		string alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
 		string p1 = alphabet.Get(Math.RandomInt(0, 25)) + alphabet.Get(Math.RandomInt(0, 25));
```

Comment:
```suggestion
		string p1 = alphabet.Get(Math.RandomInt(0, alphabet.Length() - 1)) + alphabet.Get(Math.RandomInt(0, alphabet.Length() - 1));
```

Reply:
Done in https://github.com/EveronLife/EveronLife/pull/121/commits/c3a1d7bc3e043d691cd0b3d90c1723d1fc9c63cb